torch/_dynamo/eval_frame.py

```diff
@@ -584,7 +584,7 @@ class directly; instead, use :func:`torch._export.dynamic_dim`.
 def export(
     f: Callable[..., Any],
     *args,
-    aten_graph: bool = False,
+    aten_graph: str = "none",
```

Comment:
I don't think I would have personally gone ahead and done a BC-breaking change here. But if we are in the business of making a BC-breaking change, I think we ought to also rename the keyword argument as well. It's weird and redundant to say `aten_graph="aten"`. How about `dialect` or `target_dialect`? Also cc @SherlockNoMad for opinions.

Reply:
`dialect` sounds good to me (but interested in Sherlock's opinion).

I'm also concerned about BC risk - if you're worried about changing the bool, I'm happy to leave it alone (although two bools to control the "dialect" feels bad). Fixing any internal call-sites will be annoying but doable.

Reply:
If you're willing to put in the effort, fixing the API earlier rather than later is good. I just remember I had specifically asked you to plumb this, and had been thinking you'd do the low effort thing.

Reply:
you're right - in interesting of landing sooner, I'm avoiding the API concern for now and adding an extra boolean argument. cc @andrewor14